backendAS/tables.sql

```diff
@@ -3,9 +3,8 @@ create database player;
 
 \connect player
 
-
 create table song(
-  id int serial primary key,
+  id serial primary key,
```

Comment:
Could you tell me what serial does over int serial?

Reply:
word on the street is that serial results in the automatic creation of an int or bigint column.....
so no need to specify.